test/shared/implementation/V4RouterImplementation.sol

```diff
@@ -3,19 +3,31 @@ pragma solidity ^0.8.19;
 
 import {IPoolManager} from "@uniswap/v4-core/src/interfaces/IPoolManager.sol";
 import {Currency} from "@uniswap/v4-core/src/types/Currency.sol";
-import {IERC20Minimal} from "@uniswap/v4-core/src/interfaces/external/IERC20Minimal.sol";
 import {V4Router} from "../../../src/V4Router.sol";
 import {ReentrancyLock} from "../../../src/base/ReentrancyLock.sol";
+import {SafeTransferLib} from "solmate/src/utils/SafeTransferLib.sol";
+import {ERC20} from "solmate/src/tokens/ERC20.sol";
 
 contract V4RouterImplementation is V4Router, ReentrancyLock {
+    using SafeTransferLib for *;
+
     constructor(IPoolManager _poolManager) V4Router(_poolManager) {}
 
-    function executeActions(bytes calldata params) external isNotLocked {
+    function executeActions(bytes calldata params) external payable isNotLocked {
         _executeActions(params);
     }
 
+    function executeActionsAndSweepExcessETH(bytes calldata params) external payable isNotLocked {
```

Comment:
why is SWEEP_ETH not a command? intentional? or just waiting for posm to implement it?

Reply:
Just no need for it to be inside the v4 lock! UR already has one for v2 and v3 so may as well just re-use that one outside of the lock